### ISSUE TITLE
Suggestion for already existing plugin RoleColorEverywhere

### ISSUE THREAD

**Issue by RiuKy** [Vencord/plugin-requests#758]
### Plugin Description

Add an option for the plugin RoleColorEverywhere to have an outline of chosen color (for example white or black) for the usernames in servers' lists and voice chats. For example if I have a black theme and someone has a black role, with consequently black name, I can't really understand what his name is unless I open his profile, with a white outline there would be no problem. 

example |
ㅤㅤㅤㅤv
![image](https://github.com/Vencord/plugin-requests/assets/119540878/636add9c-6dc4-45fe-822f-002fc18750af)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by nvhhr** [comment]
This can be kind of done in css using -webkit-text-stroke or multiple text-shadows but it looked terrible. Not sure if a plugin could make a better stroke. 
I came up with another solution that just brightens names:
https://github.com/nvhhr/discordcss/blob/main/snippets/namebrightener.theme.css

**Comment by korbosoft** [comment]
> This can be kind of done in css using -webkit-text-stroke or multiple text-shadows but it looked terrible. Not sure if a plugin could make a better stroke. I came up with another solution that just brightens names: https://github.com/nvhhr/discordcss/blob/main/snippets/namebrightener.theme.css
This is true, but I (personally) do agree with them that this should just be part of RoleColorEverywhere. I have seen black get used as the "Muted" color loads of times in the past, and i feel this would help make things more readable when using the plugin.

**Comment by korbosoft** [comment]
i give up im not meant for making vencord plugins this is extremely hard
